packages/components/src/draggable/index.native.js

```diff
@@ -142,7 +142,7 @@ const Draggable = ( {
 
 	const providerValue = useMemo( () => {
 		return { panGestureRef, isDragging, isPanActive, draggingId };
-	}, [] );
+	}, [ isDragging, isPanActive, draggingId ] );
```

Comment:
Thanks @fluiddot!

Reply:
Thank you both!

Maybe we could leave an inline comment in code, so that the next time folks look at this line of code they have more context about `isDragging`, `isPanActive` and `draggingId` ?

Reply:
Yep good point, we could leave a comment when using the shared value objects in hooks to outline that they are not required to be added in the dependencies array 👍 .

Reply:
Tiny follow-up opened in #41658